queries.sql

```diff
@@ -1,3 +1,67 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+/*id |  name   | date_of_birth | escape_attempts | neutered | weight_kg
+----+---------+---------------+-----------------+----------+-----------
+  1 | Agumon  | 2020-02-03    |               0 | t        |     10.23
+  2 | Gabumon | 2018-11-15    |               2 | t        |       8.0
+  4 | Devimon | 2017-05-12    |               5 | t        |      11.0
+(3 rows) */
+
+--List the name of all animals born between 2016 and 2019.
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-1-1' AND '2019-1-1';
```

Comment:
- [ ] On line 12, instead of using '2019-1-1', please use '2019-31-12' so the December month is also selected for that year.  

That way, the data will include that month 💪